packaging/technical-addon/Splunk_TA_otel/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
-## Unreleased
+## v1.6.0
```

Comment:
It feels wrong doing this by hand here since this is not how it is done for the other artifacts coming from the repo, but I'm following the pattern in the last PR.

Reply:
Not really related to your comment, but we can keep the unreleased header. (That's what we've done historically with the top-level CHANGELOG)